spec/takeaway_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:menu) { double :menu, content: {"Fish and chips" => 7, "Steak and chips" => 10,
```

Comment:
Line is too long. [86/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space inside { missing.